forgerock-auth/src/main/java/org/forgerock/android/auth/callback/DeviceBindingCallback.kt

```diff
@@ -139,83 +161,110 @@ open class DeviceBindingCallback: AbstractCallback {
      *
      * @param context  The Application Context
      * @param listener The Listener to listen for the result
-     * @param authInterface Interface to find the Authentication Type
+     * @param deviceAuthenticator Interface to find the Authentication Type
      * @param encryptedPreference Persist the values in encrypted shared preference
      */
+    @OptIn(ExperimentalTime::class)
```

Comment:
do we need this @OptIn(ExperimentalTime::class) ? i think we can directly use this api to avoid this , can you check ?

public suspend fun <T> withTimeout(timeMillis: Long, block: suspend CoroutineScope.() -> T): T {

Reply:
We can if using the timMillis, but using the duration required the annotation. Any drawback using the annotation?